src/renderer/components/Player.tsx

```diff
@@ -1,23 +1,25 @@
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
 import {HotKeys} from 'react-hotkeys';
-
 import recursiveReaddir from 'recursive-readdir';
+import fs from 'fs'
+const animated : Function = require('animated-gif-detector');
```

Comment:
Please use `import`, it should work the same. You'll just need to add a line to `src/declaration.d.ts`: `declare module animated-gif-detector`

Reply:
I tried using `import animated from 'animated-gif-detector';` but the compiler kept complaining that `animated` wasn't typed 🤦‍♀️ So just used that as a workaround

Reply:
Yeah, that's what gets fixed when you do the `declaration.d.ts` thing

Reply:
Cool!  I didn't know how to do that, ty for including in readme.md